code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -54,6 +54,43 @@
 	tastes = list("fish" = 1, "chips" = 1)
 	foodtype = MEAT | VEGETABLES | FRIED
 
+/obj/item/reagent_containers/food/snacks/sushie_basic
+	name = "Funa Hosomaki"
+	desc = "A small cylindrical filled with rice and fish."
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_basic"
+	bonus_reagents = list("vitamin" = 2)
+	list_reagents = list("nutriment" = 4)
+	bitesize = 1
+	filling_color = "#F2EEEA" //rgb(242, 238, 234)
+	tastes = list("fish" = 1, "rice" = 1, "salt" = 1)
+	foodtype = MEAT | VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/sushie_adv
+	name = "Funa Nigiri"
+	desc = "A pice of carp lightly placed on some rice."
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_adv"
+	bonus_reagents = list("vitamin" = 2)
+	list_reagents = list("nutriment" = 6)
+	bitesize = 1
+	filling_color = "#F2EEEA" //rgb(242, 238, 234)
+	tastes = list("fish" = 1, "rice" = 1, "salt" = 1)
+	foodtype = MEAT | VEGETABLES
+
+/obj/item/reagent_containers/food/snacks/sushie_pro
+	name = "Funa Nigiri"
+	desc = "A well prepared pice of the best of the carp fillet placed on rice. Looks fancy and fresh!"
+	icon = 'modular_citadel/icons/obj/food/food.dmi'
+	icon_state = "sushie_pro"
+	bonus_reagents = list("nutriment" = 2, "vitamin" = 2)
+	list_reagents = list("nutriment" = 8, "vitamin" = 2)
+	bitesize = 1
```

Comment:
five carp fillets for 1 bite and 10 nutriment. 

seems uh, a bit off?

Reply:
1 bite means it takes 1 of each nutriment per bite
Its not ment to be eaten all at once!

Reply:
Bite size still needs to be changed. The carp fillets, on their own, are a size 3 bite of 7-9. This is way smaller and counter-intuitive. Considering the fact that sushi is a bite-size piece of food, this makes no sense.